app/services/platform-apps/api/modules/scene-transitions.ts

```diff
@@ -109,6 +111,80 @@ export class SceneTransitionsModule extends Module {
     throw new Error('Not Implemented');
   }
 
+  /**
+   * Get a list of transitions
+   *
+   * This includes all transitions the user has set up in SLOBS.
+   * For transitions managed by this App use `getAppTransitions`.
+   *
+   * @see {getAppTransitions}
+   */
+  @apiMethod()
+  async getTransitions(_ctx: IApiContext): Promise<Array<ITransition>> {
+    return this.transitionsService.state.transitions;
+  }
+
+  /**
+   * Get a list of transitions belonging to this App.
+   *
+   * @param ctx API context
+   * @return A list of Transition objects belonging to the current App
+   */
+  @apiMethod()
+  async getAppTransitions(ctx: IApiContext): Promise<Array<ITransition>> {
+    return Object.keys(this.transitionsService.obsTransitions)
```

Comment:
`this.transitionsService.obsTransitions` is really just meant to be a lookup table for OBS object references.  While this technically works, it would be more correct to rely on what's stored in state here:
```TypeScript
this.transitionsService.transitions.filter(...)
```
This should also be more efficient since you don't need to map to transitions at the end.

Reply:
`transitions` is not exposed (at least in my testing), so I'd have to reach out into `state.transitions` which looked like a worse option, `obsTransitions` is. I'm happy to use that if we're good with the cosmetic loss of encapsulation.

Reply:
Ah right.  We can just expose a getter for cosmetic reasons if we prefer.  I kind of go back and forth on whether encapsulating state is beneficial.  As long as a service has complete ownership over mutations and when they occur, it's OK for other parts of the code to get read-only access to that state.  Getters can be used to provide useful "views" into that state, but it's not essential.

Reply:
I've just used the other method. Have to pass `ctx` but it's not the end of the world. Using `transitions` is indeed more efficient, thought there's still an extra lookup for `propertiesManager`.